postman/schemas/paths/component/data-science-by-id.yaml

```diff
@@ -0,0 +1,64 @@
+# /component/data-science/{id}
+get:
+    summary: Get all information about a data-science configuration.
+    operationId: getDataScienceConfiguration
+    parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+            type: string
+            format: uuid
+    responses:
+        "200":
+            description: Successfull operation
+            content:
+                application/json:
+                    schema:
+                        $ref: "../../schemas/DataScience.yaml#/components/schema/DataScienceConfiguration"
+        "401":
+            description: API token missing
+        "404":
+            description: Id not found
+delete:
+    summary: Delete a data-science configuration.
+    operationId: deleteDataScienceConfiguration
+    parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+              type: string
+              format: uuid
+    responses:
+        "204":
+            description: Deleted data-science configuration
+        "401":
+            description: API token missing
+        "404":
+            description: Id not found
+        
+patch:
+    summary: Edit the a data-science configuration.
+    operationId: updateDataScienceConfiguration
+    parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+                type: string
+                format: uuid
+    requestBody:
+        content:
+            application/json:
+                schema:
+                    $ref: "../../schemas/DataScience.yaml#/components/schema/DataScienceConfiguration"
+    responses:
+        "204":
+            description: Updated data-science configuration
+        "400":
+            description: Update not possible, you can't update the id
```

Comment:
When is this the case?

Reply:
Good question 👍 The first idea was, that you can specify the id by your own. But I'm not a fan of this idea anymore. I was thinking about not allowing to define and update the id. I'll fix that! 🤝 Thank you!